website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+    We create a `proxy` that connects to the ClusterLink gateway and establishes a reverse tunnel to allow other clients to connect.
+    We also create an FRP `visitor` that specifies which other peers this client wants to connect to (you need to create a visitor for every peer you want to connect).
+
+4. Create a K8s service that connects to the FRP client `visitor`, allowing ClusterLink to connect to it:
+
+    *Client cluster*:
+
+    ```sh
+    echo '
+    apiVersion: v1
+    kind: Service
+    metadata:
+        name: server-peer-clusterlink
+        namespace: clusterlink-system
+    spec:
+        type: ClusterIP
+        selector:
+            app: frpc
+        ports:
+            - port: 6002
+              targetPort: 6002
+    ' | kubectl apply -f -
+     ```
+
+    *Server cluster*:
+
+    ```sh
+    echo '
+    apiVersion: v1
+    kind: Service
+    metadata:
+        name: client-peer-clusterlink
+        namespace: clusterlink-system
+    spec:
+        type: ClusterIP
+        selector:
+            app: frpc
+        ports:
+            - port: 6001
+              targetPort: 6001
+    ' | kubectl apply -f -
+     ```
+
+4. Create FRP client deployment for each cluster:
+
+    *Client cluster*:
+
+    ```sh
+    echo "
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+        name: frpc
+        namespace: clusterlink-system
+    spec:
+        replicas: 1
+        selector:
+            matchLabels:
+                app: frpc
+        template:
+            metadata:
+                labels:
+                    app: frpc
+            spec:
+                containers:
+                    - name: frpc
+                      image: snowdreamtech/frpc
+                      volumeMounts:
+                        - name: frpc-config-volume
+                          mountPath: /etc/frp
+                volumes:
+                  - name: frpc-config-volume
+                    configMap:
+                        name: frpc-config
+                restartPolicy: Always
+        " | kubectl apply -f -
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    echo "
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+        name: frpc
+        namespace: clusterlink-system
+    spec:
+        replicas: 1
+        selector:
+            matchLabels:
+                app: frpc
+        template:
+            metadata:
+                labels:
+                    app: frpc
+            spec:
+                containers:
+                    - name: frpc
+                      image: snowdreamtech/frpc
+                      volumeMounts:
+                        - name: frpc-config-volume
+                          mountPath: /etc/frp
+                volumes:
+                  - name: frpc-config-volume
+                    configMap:
+                        name: frpc-config
+                restartPolicy: Always
+        " | kubectl apply -f -
+    ```
+
+## Create Peer CRDs
```

Comment:
I think you should be more explicit on how the Peer CRD is defined here to reference the FRP service and how it works at a high level.